website/public/demo/error-message.mjs

```diff
@@ -0,0 +1,57 @@
+function showErrorMessageBox(mark) {
+  const div = createErrorBox(
+    mark,
+    mark.attributes["data-message"].value,
+    mark.attributes["data-code"].value,
+    mark.attributes["data-severity"].value
+  );
+  let body = document.querySelector("body");
+  body.appendChild(div);
+}
+
+function createErrorBox(markedElement, errorMessage, code, severity) {
+  // TODO: Change background color based of the severity
+  let div = document.createElement("div");
+  const { bottom, left, height } = markedElement.getBoundingClientRect();
+  div.setAttribute("id", "error-message-box");
+  div.innerText = `${code} - ${errorMessage}`;
+  div.setAttribute(
+    "style",
+    `position: fixed; 
+    top: ${Math.trunc(bottom)}px; 
+    left: ${Math.trunc(left)}px`
+  );
```

Comment:
Nit: Isn't it more "clean" to use `div.style.position = 'fixed';`, etc.? I'm not up to date with the hottest ways to set inline styles, so maybe what you have is recommended.